plugins/redis32/src/main/resources/redis1.conf

```diff
@@ -0,0 +1,6 @@
+bind test1
```

Comment:
Uh uh uh, I hope it's not necessary to put in several files and set the bind address to `testX` (which is not defined in RG anywhere). If you need the servers to have separate configs, RG should copy the file and overwrite bind address, e.g. by adding a `bindInterface` property (or letting it to bind to all interfaces on given machine).